managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -463,6 +474,10 @@ public Result create(UUID customerUUID) {
       LOG.info("Create for {}.", customerUUID);
       // Get the user submitted form data.
       formData = (ObjectNode) request().body().asJson();
+      if (validateEncryption(formData))
+      {
+        return ApiResponse.error(BAD_REQUEST, "Node to node encryption must required to enable for client to node encryption.");
+      }
```

Comment:
For now, i just have added validation for creating api let me know if we need to add any other api.

Reply:
Please verify that UI processes the possible error correctly. Often it doesn't.

Reply:
This specific scenario has been handled in the Front end.